packages/protocol/contracts/governance/DowntimeSlasher.sol

```diff
@@ -5,22 +5,33 @@ import "openzeppelin-solidity/contracts/math/SafeMath.sol";
 
 import "./SlasherUtil.sol";
 
+/// @dev Version: 2.0.0.0
```

Comment:
yes it does :(

Reply:
The 1.0.0.0 version didn't worked, so I think that we are going to deploy a new proxy (just in case), add this new version, and change the registry to point to that pair, instead of the old one